### DISCUSSION TITLE
Footer

### DISCUSSION THREAD

**Discussion by jrbarnes9** [ONSdigital/design-system#172] [Feedback]
Use this place to discuss the [footer component](https://ons-design-system.netlify.com/components/footer) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by bameyrick** [comment]
The top margin from the footer should be removed and added to the [page template](https://github.com/ONSdigital/pattern-library-v2/issues/195) instead.

**Comment by bameyrick** [comment]
The layout of the columned footer is broken on desktop:

![Screenshot 2019-06-18 at 11 42 52](https://user-images.githubusercontent.com/8447446/59676017-78eb2180-91be-11e9-8485-b7202924c466.png)

**Comment by jrbarnes9** [comment]
`footer__hr` is too dark.  Suggest using a new `$color-grey-3` (`#ACACAC`)

**Comment by jrbarnes9** [comment]
Footer logo spacing issue 
![Screenshot 2020-10-05 at 16 26 09](https://user-images.githubusercontent.com/43346934/95099263-929fd780-0727-11eb-9dab-7e5650a7b9b9.png)

**Comment by trevorsaint** [comment]
Need to have the ability to show an OS disclaimer in the footer. Something similar to the example included.

<img width="1104" alt="footer_example" src="https://user-images.githubusercontent.com/4989027/96145249-edda8280-0efc-11eb-93e5-8837b9fce7be.png">

**Comment by trevorsaint** [comment]
We seem to be missing margin right on the OGL logo.

<img width="372" alt="Screenshot 2020-10-26 at 11 29 41" src="https://user-images.githubusercontent.com/4989027/97167406-a070d780-177e-11eb-8241-0c9d146fcc2a.png">

**Comment by jrbarnes9** [comment]
Incorrect styling on footer button

Should sit on same background colour as header and use the exit icon by default

From this...
![Screenshot 2020-10-27 at 08 02 39](https://user-images.githubusercontent.com/43346934/97273242-01ee8000-182b-11eb-89ac-11c41ca4588a.png)
To this...
<img width="339" alt="Screenshot 2020-10-27 at 08 02 34" src="https://user-images.githubusercontent.com/43346934/97273246-02871680-182b-11eb-985d-fd72347d8cb9.png">

**Comment by jrbarnes9** [comment]
Footer should use a fluid layout, allowing links to wrap in a row

From this...
![Screenshot 2020-10-27 at 08 12 31](https://user-images.githubusercontent.com/43346934/97274706-ee441900-182c-11eb-97d9-f7f7ce3d8ec0.png)

To this...
![Screenshot 2020-10-27 at 08 18 39](https://user-images.githubusercontent.com/43346934/97274784-09af2400-182d-11eb-8a37-dc482a414d4c.png)

**Comment by jrbarnes9** [comment]
Links in example footer with `newTabWarning` don't contain appropriate attributes (e.g. `target`)

**Comment by jrbarnes9** [comment]
Footer OGL in wrong position when used in 2 row layout

Should be displayed after the legal footer links row. 

![Screenshot 2021-06-15 at 15 44 38](https://user-images.githubusercontent.com/43346934/122074013-c5d75200-cdf0-11eb-85c8-e69f03252b49.png)

**Reply by jrbarnes9** [reply]
#1557 raised

**Comment by jrbarnes9** [comment]
Footer row links need more spacing when wrapped.

Current spacing...
![Screenshot 2022-01-21 at 13 40 47](https://user-images.githubusercontent.com/43346934/150538177-a3b68f5b-fc8b-4a29-98ed-451e9fd77a5d.png)

Should match spacing in list columns...
<img width="254" alt="Screenshot 2022-01-21 at 13 50 42" src="https://user-images.githubusercontent.com/43346934/150538273-a0da1abf-4cd8-41f0-b70d-1e79b9300670.png">

**Reply by jrbarnes9** [reply]
Raised #1860

**Comment by jrbarnes9** [comment]
https://github.com/ONSdigital/design-system/discussions/160#discussioncomment-3536864

**Reply by jrbarnes9** [reply]
Raised #2434

**Comment by jrbarnes9** [comment]
`rows` missing from transactional footer example

**Reply by jrbarnes9** [reply]
Raised #2555
